tests/test_general_loader.py

```diff
@@ -0,0 +1,163 @@
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.ibw",
+            "HeightTracee",
+            False,
+            "Extracted image",
+            id="'.ibw' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.ibw",
+            "Hight",
+            True,
+            "'Hight' not in .ibw channel list: ['HeightTracee', 'HeightRetrace', 'ZSensorTrace', 'ZSensorRetrace', "
+            "'UserIn0Trace', 'UserIn0Retrace', 'UserIn1Trace', 'UserIn1Retrace']",
+            id="'.ibw' channel not found.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.jpk",
+            "height_trace",
+            False,
+            "Extracted image",
+            id="'.jpk' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.jpk",
+            "might_base",
+            True,
+            "'might_base' not in .jpk channel list: {'height_retrace': 1, 'measuredHeight_retrace': 2, "
+            "'amplitude_retrace': 3, 'phase_retrace': 4, 'error_retrace': 5, 'height_trace': 6, "
+            "'measuredHeight_trace': 7, 'amplitude_trace': 8, 'phase_trace': 9, 'error_trace': 10}",
+            id="'.jpk' channel not found.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.spm",
+            "Height",
+            False,
+            "Extracted channel Height",
+            id="'.spm' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.spm",
+            "Force",
+            True,
+            "'Force' not in .spm channel list: ['Height Sensor', 'Peak Force Error', 'DMTModulus', 'LogDMTModulus', "
+            "'Adhesion', 'Deformation', 'Dissipation', 'Height']",
+            id="'.spm' channel not found.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.stp",
+            "",
+            False,
+            "Extracted image",
+            id="'.stp' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.top",
+            "",
+            False,
+            "Extracted image",
+            id="'.top' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0_1.topostats",
+            "image",
+            False,
+            "Extracted .topostats dictionary.",
+            id="'.topostats' success.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0_1.topostats",
+            "hgjswbweongp",
+            True,
+            "'hgjswbweongp' not in available image keys: ['image']",
+            id="'.topostats' channel not found.",
+        ),
+        pytest.param(
+            RESOURCES / "sample_0.xxx",
+            "NotAChannel",
+            True,
+            "File type '.xxx' is not currently handled by AFMReader.",
+            id="'.xxx' unsupported filetype.",
+        ),
+    ],
+)
+def test_load(caplog: pytest.LogCaptureFixture, filepath: Path, channel: str, error: bool, message: str):
```

Comment:
```suggestion
def test_load(caplog: pytest.LogCaptureFixture, filepath: Path, channel: str, error: bool, message: str) -> None:
```

I'm being pedantic, sorry